lib/engine/game/g_1837/step/token.rb

```diff
@@ -27,7 +27,7 @@ def can_place_token?(entity)
             # Cheaper to do the graph first, then check affordability
             current_entity == entity &&
               !(token = entity.next_token).nil? &&
-              @game.graph.can_token?(entity) &&
+              @game.token_graph.can_token?(entity) &&
```

Comment:
`@game.token_graph_for_entity(entity).can_token?(entity)[hex]`

Reply:
Updated (even though it does the same thing).

Reply:
It does today, but the implementations can diverge later and I'd rather have the code use the one its supposed to.